### PR TITLE
Mollie Components: Clarify error handling examples on the client-side

### PR DESCRIPTION
**Pull request by pabloroman**
Ideally we should add more information about the different error messages we return @mattijsbliek . I can add those if you give me a list.

### CODE REVIEW

File: source/guides/mollie-components/overview.rst [hunk 1]
```diff
@@ -159,16 +159,20 @@ You can then place the ``cardToken`` in a hidden input to submit it to your back
 
      const { token, error } = await mollie.createToken();
 
-     // Add token to the form
-     const tokenInput = document.createElement('input');
-     tokenInput.setAttribute('type', 'hidden');
-     tokenInput.setAttribute('name', 'cardToken');
-     tokenInput.setAttribute('value', token);
+     if (error) {
+       // Something wrong happened while creating the token. Handle this situation gracefully.
```
Comment (mattijsbliek) [inline source/guides/mollie-components/overview.rst]: Can you early return so you don’t need the `else`? All good otherwise 🙌

**Comment by willemstuursma** [comment]
An example thing that can happen is: the card is not supported (e.g. Discover card). This should be reflected back to the shopper, maybe even automatically.

File: source/guides/mollie-components/overview.rst [hunk 2]
```diff
@@ -159,16 +159,21 @@ You can then place the ``cardToken`` in a hidden input to submit it to your back
...
+       return;
+     } else {
```
Comment (mattijsbliek) [inline source/guides/mollie-components/overview.rst]: You can now get rid of this `else` statement :)

**Comment by mattijsbliek** [comment]
> An example thing that can happen is: the card is not supported (e.g. Discover card). This should be reflected back to the shopper, maybe even automatically.

This is already handled automatically by Mollie Components. The only thing we don’t handle is Mollie supporting the card, but it not being enabled on the merchant’s account (e.g. Amex being turned off).

File: source/guides/mollie-components/overview.rst (hunk 2 above)
Comment (pabloroman) [reply source/guides/mollie-components/overview.rst]: Shame on me 😱 Sorted!

Comment (mattijsbliek) [reply source/guides/mollie-components/overview.rst]: For sure!

**Pull request merged**